### ISSUE TITLE
addClassNames could return the element itself

### ISSUE THREAD

**Issue by dyoxyne** [WizUI/WuiDom#55]
It could be nice if addClassNames could return the element itself.

`btnCreate = row.appendChild(new Button('CREATE')).addClassNames('col-lg-3');`

**Comment by ronkorving** [comment]
Agreed, the code above currently wouldn't work, because `addClassNames` doesn't return the created button. So people often end up passing class names to constructors instead (to keep the call-site short). The extra code in the constructor is useless, and it doesn't really make the user APIs any cleaner.
